prime-angular-frontend/src/app/modules/adjudication/shared/components/site-registration-actions/site-registration-actions.component.ts

```diff
@@ -20,6 +20,7 @@ export class SiteRegistrationActionsComponent implements OnInit {
   @Output() public escalate: EventEmitter<number>;
   @Output() public delete: EventEmitter<{ [key: string]: number }>;
   @Output() public enableEditing: EventEmitter<number>;
+  @Output() public toggleFlagSite: EventEmitter<{ siteId: number, flagged: boolean }>;
```

Comment:
for naming convention, the site actions are assumed to be on the site, as seen with events emitted above. So this could just be the verb.